### PR TITLE
[WIP] add more answers related to isolation status, fixes #262

### LINKED ISSUE #262: Revised options under "I am currently" in form
I think we should have these options under "I am currently":

* Not in isolation
* In isolation due to COVID-19 infection
* In isolation due to a recent international travel
* In isolation due to contact with an individual who has been exposed to the coronavirus

### PR DESCRIPTION
**Pull request by fossecode**
Need translations and confirmations for:
- [x] Danish
- [x] English
- [x] Spanish (Colomibia)
- [x] Spanish (Mexico)
- [x] French
- [x] Italian
- [x] Dutch
- [x] Norwegian
- [x] Slovak
- [x] Turkish

### CODE REVIEW

**Comment by fossecode** [comment]
@bertrandlalo @zerbfra @adriaanvanrossum @michaelmcmillan @btutal @alny @julianbetancourt @alexalmadav @sbocinec 

Can you please paste translations for your countries?

**Comment by zerbfra** [comment]
Going to check in an hour

File: app/locales/fr-fr.json [hunk 1]
```diff
@@ -174,5 +174,7 @@
   "Close": "Fermé",
   "Contribute now": "Participer",
   "Help us uncover and predict the corona spread in your area": "Aidez-nous à <span>estimer</span> et <span>prédire</span> la propagation du corona virus dans votre région",
-  "Numbers": "Chiffres"
+  "Numbers": "Chiffres",
+  "In isolation due to orders from the government": "In isolation due to orders from the government",
+  "In isolation due to COVID-19 infection": "In isolation due to COVID-19 infection"
```
Comment (bertrandlalo) [inline app/locales/fr-fr.json]: "In isolation due to orders from the government": "isolé(e) en raison des ordres du gouvernement",
  "In isolation due to COVID-19 infection": "isolé(e) car vous êtes infecté(e) par le COVID-19"

**Review by bertrandlalo — COMMENTED** [review]
Nice, I added the french translation. 
Also, should'nt we mention it in part 'Clarifications data basis' where you added the date where changes in the questionnaire occured?

File: app/locales/fr-fr.json (hunk 1 above)
Comment (bertrandlalo) [reply app/locales/fr-fr.json]: @fossecode 
Is there something else?

**Comment by tarasmatsyk** [comment]
@fossecode here is a part in Ukrainian:
```
  "Some questions and alternatives were not a part from the beginning:": "Деякі питання і альтернативні відповіді не були частиною опитування:",
  "March 27": "Березень 27"
```

**Comment by julianbetancourt** [comment]
## es-co

```
"In isolation due to orders from the government": "En aislamiento por orden del gobierno",
"In isolation due to COVID-19 infection": "En aislamiento por contagio de COVID-19",
"Some questions and alternatives were not a part from the beginning:": "Algunas preguntas no fueron incluidas desde el inicio:",
"March 27": "Marzo 27"
``` 

@fossecode

**Comment by sbocinec** [comment]
SK version PR landed here https://github.com/BustByte/coronastatus/pull/381

**Comment by zerbfra** [comment]
Here it is the italian version:
```
  "In isolation due to orders from the government": "In quarantena per ordine del Governo Italiano",
  "In isolation due to COVID-19 infection": "In isolamento per infezione da COVID-19 accertata",
  "Some questions and alternatives were not a part from the beginning:": "Alcune domamdne non erano parte del questionario dall'inizio:",
```

Thanks

File: app/locales/es-mx.json [hunk 2]
```diff
@@ -1,180 +1,183 @@
...
+  "I currently smoke": "Actualmente fumo",
+  "I used to smoke": "Solia fumar",
+  "I have never smoked": "Nunca he fumado",
+  "What is your current body temperature? (Celcius)": "Cual es tu temperatura corporal en grados Celsius?",
+  "This field is optional": "Este campo es opcional",
+  "Close": "Cerrar",
+  "Contribute now": "Contribuir ahora",
+  "Help us uncover and predict the corona spread in your area": "Ayuda a <span>descurbrir</span> y <span>predecir</span> la transmision del coronavirus en tu region",
+  "Numbers": "Numeros",
+  "In isolation due to orders from the government": "In isolation due to orders from the government",
```
Comment (alexalmadav) [inline app/locales/es-mx.json]: can you change this to "En aislamiento por ordenes del gobierno"

File: app/locales/es-mx.json [hunk 3]
```diff
@@ -1,180 +1,183 @@
...
+  "In isolation due to COVID-19 infection": "In isolation due to COVID-19 infection",
```
Comment (alexalmadav) [inline app/locales/es-mx.json]: "En aislamiento por infeccion de COVID-19"

File: app/locales/es-mx.json [hunk 4]
```diff
@@ -1,180 +1,183 @@
...
+  "Some questions and alternatives were not a part from the beginning:": "Some questions and alternatives were not a part from the beginning:",
```
Comment (alexalmadav) [inline app/locales/es-mx.json]: "Algunas preguntas y sus respuestas no formaban parte del formulario original"

**Review by alexalmadav — CHANGES_REQUESTED** [review]

File: app/locales/tr.json [hunk 5]
```diff
@@ -175,5 +175,9 @@
   "Close": "Kapat",
   "Contribute now": "Şimdi destek ol",
   "Help us uncover and predict the corona spread in your area": "Bölgendeki corona yayılımını analiz edebilmemiz için bize destek ol",
-  "Numbers": "Sayılar"
+  "Numbers": "Sayılar",
+  "In isolation due to orders from the government": "In isolation due to orders from the government",
```
Comment (sariug) [inline app/locales/tr.json]: "In isolation due to orders from the government": "Hükümet tarafından alınan kararlar dolayısıyla karantinada",
  "In isolation due to COVID-19 infection": "COVID-19 bulaşması yüzünden karantinada",
  "Some questions and alternatives were not a part from the beginning:": "Bazı sorular ve alternatifler en başından beri yoktu:",

@btutal  can proof read and edit the file as a contributor.

Comment (btutal) [reply app/locales/tr.json]: Thanks @sariug , I've just commited Turkish trnslations

**Review by btutal — APPROVED** [review]
Turkish is completed

**Comment by michaelmcmillan** [comment]
**do we know anyone who can speak norwegian here????** 😟 
Screenshot 2020-03-26 at 22 10 08

**Comment by adriaandotcom** [comment]
They are always soo slowwww

**Pull request merged**